gson/src/main/java/com/google/gson/internal/bind/TypeAdapters.java

```diff
@@ -693,6 +678,104 @@ public void write(JsonWriter out, Locale value) throws IOException {
     }
   };
 
+  public static final TypeAdapter<LocalDateTime> LOCAL_DATE_TIME = new TypeAdapter<LocalDateTime>() {
```

Comment:
These aren't being registered anywhere so they're not really useful. Moreover, we cannot register them automatically because the library has to support Java 6 and 7.